github/issues.go

```diff
@@ -54,6 +55,7 @@ type Issue struct {
 	// TextMatches is only populated from search results that request text matches
 	// See: search.go and https://developer.github.com/v3/search/#text-match-metadata
 	TextMatches []TextMatch `json:"text_matches,omitempty"`
+	NodeID      *string     `json:"node_id,omitempty"`
```

Comment:
Also here, move `NodeID` field up to be with all other fields, rather than with `TextMatches`.